packages/experiments/src/components/CollapsibleSection/CollapsibleSection.types.ts

```diff
@@ -42,11 +42,15 @@ export type ICollapsibleSectionControlledProps = IStyleableComponentProps<
 
 export type ICollapsibleSectionViewProps = Pick<ICollapsibleSectionProps, 'titleAs' | 'titleProps'> &
```

Comment:
the typings here bewilder me.

I think I understand what's going on, but you might want to explain in a comment. (you're stripping off defaultCollapsed, I believe? will every component need to do this?)

Reply:
This is not particular to createComponent, we've been doing with our newer JS styled components in OUFR. This is the equivalent of defining StyleProps. When Style/View props borrow props from the component props, they should be bound this way instead of loosely coupled. It'll improve ease of maintenance by catching binding issues when component props change.